library/std/src/keyword_docs.rs

```diff
@@ -993,7 +990,6 @@ mod mod_keyword {}
 /// For more information on the `move` keyword, see the [closure]'s section
 /// of the Rust book or the [threads] section
 ///
-/// [`Fn` trait]: ../std/ops/trait.Fn.html
```

Comment:
This will give a broken link.
```suggestion
 /// [`Fn` trait]: crate::ops::Fn
```